board/e5_dev_board/src/main.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/lora.h>
+#include <errno.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/kernel.h>
+#include <stdio.h>
+#include <stdlib.h>
+#define DEFAULT_RADIO_NODE DT_ALIAS(lora0)
+BUILD_ASSERT(DT_NODE_HAS_STATUS(DEFAULT_RADIO_NODE, okay),
+	     "No default LoRa radio specified in DT");
+
+#define MAX_DATA_LEN 255
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(lora_receive);
+
+// configure reception
+int configureRx(const struct device *const lora_dev)
+{
+	struct lora_modem_config config;
+	config.frequency = 865100000;
+	config.bandwidth = BW_125_KHZ;
+	config.datarate = SF_10;
+	config.preamble_len = 8;
+	config.coding_rate = CR_4_5;
+	config.iq_inverted = false;
+	config.public_network = false;
+	config.tx_power = 14;
+	config.tx = false;
+
+	int ret = lora_config(lora_dev, &config);
+	if (ret < 0) {
+		LOG_ERR("LoRa config failed");
+	}
+	return ret;
+}
+
+// configure transmission
+int configureTx(const struct device *const lora_dev)
+{
+	struct lora_modem_config config;
+	config.frequency = 865100000;
+	config.bandwidth = BW_125_KHZ;
+	config.datarate = SF_10;
+	config.preamble_len = 8;
+	config.coding_rate = CR_4_5;
+	config.iq_inverted = false;
+	config.public_network = false;
+	config.tx_power = 14;
+	config.tx = true;
+
+	int ret = lora_config(lora_dev, &config);
+	if (ret < 0) {
+		LOG_ERR("LoRa config failed");
+	}
+	return ret;
+}
+
+void recev_msg(const struct device *dev, uint8_t *data, int16_t rssi, int8_t snr){
+	int len;
+	configureRx(dev);
+	LOG_INF("Recive data");
+	len = lora_recv(dev, data, MAX_DATA_LEN, K_FOREVER, &rssi, &snr);
+	if (len < 0) {
+		LOG_ERR("LoRa receive failed");
+		return 0;
+	}
+
+	LOG_INF("Received data: %s (RSSI:%ddBm, SNR:%ddBm)", data, rssi, snr);
+}
+
+void send_msg(const struct device *dev, uint8_t *data, int16_t rssi, int8_t snr){
+	char data_send[MAX_DATA_LEN] = {0};
+	int ret;
+	LOG_INF("Sending data");
+	int convertedValue = atoi(data);
```

Comment:
please use the same naming rule for variables most of the zephyr projects follow the camel_case rule, change the `convertedValue` to converted_value`